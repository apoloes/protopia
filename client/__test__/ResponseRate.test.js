import { shallowMount } from '@vue/test-utils'
import ResponseRate from '../src/components/ResponseRate'

describe('ResponseRate.vue', () => {
  it('renders item in props.items', () => {
    const wrapper = shallowMount(ResponseRate);
    expect(wrapper.vm.$options._propKeys).toHaveLength(1)
  })


  it('matches snapshot', () => {
    const wrapper = shallowMount(ResponseRate);
    expect(wrapper.html()).toMatchSnapshot();
  })
})
