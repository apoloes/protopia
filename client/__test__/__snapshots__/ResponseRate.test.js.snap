// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ResponseRate.vue matches snapshot 1`] = `undefined`;
